presto-product-tests/bin/run_on_docker.sh

```diff
@@ -102,7 +102,7 @@ function stop_docker_compose_containers() {
     stop_application_runner_containers ${ENVIRONMENT}
 
     # stop containers started with "up", removing their volumes
-    environment_compose down -v
+    environment_compose down -v || true
```

Comment:
I'd say yes - we had cases where all the tests passed, yet docker-compose got a timeout when trying to communicate with the sqlserver container. Since the problem is hard to reproduce locally, we've no idea how to approach a fix, and it does not affect the tests themselves - I'd stick with it.